### PR TITLE
Add journal errors regarding #14818 to config

### PR DESCRIPTION
**Pull request by asdil12**
See also:
* https://bugzilla.suse.com/show_bug.cgi?id=1199163
* #14818

### CODE REVIEW

**Comment by asdil12** [comment]
openqa-clone-custom-git-refspec https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14821 https://openqa.suse.de/tests/8676748
Github oauth token provided, performing authenticated requests
Created job #8682285: sle-15-SP4-JeOS-for-RaspberryPi-aarch64-Build1.134-jeos-realhw-RPi@RPi4 -> https://openqa.suse.de/t8682285

**Review by jlausuch — APPROVED** [review]

**Review by jlausuch — APPROVED** [review]
Looks like it doesn't match: 
`Directory 'fef00700.hdmi' with parent 'vc4-hdmi-0' already present!`

https://openqa.suse.de/tests/8682285#step/journal_check/10

File: data/journal_check/bug_refs.json [hunk 1]
```diff
@@ -342,5 +342,12 @@
             "microos":  ["Tumbleweed"]
         },
         "type": "ignore"
+    },
+    "bsc#1199163": {
+        "description": "Directory '(3f902000|fef05700|fef00700).hdmi' with parent 'vc4-hdmi' already present",
```
Comment (jlausuch) [inline data/journal_check/bug_refs.json:347]: can you try
```suggestion
        "description": "Directory '(3f902000|fef05700|fef00700).hdmi' with parent 'vc4-hdmi.* already present",
```

**Comment by asdil12** [comment]
openqa-clone-custom-git-refspec https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14821 https://openqa.suse.de/tests/8676748
Github oauth token provided, performing authenticated requests
Created job #8682366: sle-15-SP4-JeOS-for-RaspberryPi-aarch64-Build1.134-jeos-realhw-RPi@RPi4 -> https://openqa.suse.de/t8682366

**Review by jlausuch — APPROVED** [review]
LGTM

**Pull request merged**